flask_potion/__init__.py

```diff
@@ -102,10 +102,16 @@ def _init_app(self, app):
         """
         app.config.setdefault('POTION_MAX_PER_PAGE', 100)
         app.config.setdefault('POTION_DEFAULT_PER_PAGE', 20)
+        decorate_schema = app.config.setdefault('POTION_DECORATE_SCHEMA', True)
```

Comment:
If the default is `True`, this will be a breaking change, in which case it should be implemented with purpose. Having just the main schema not decorated isn't very helpful. The setting should stop all schema endpoints from being decorated if `False`. A better name then is `'POTION_DECORATE_SCHEMA_ENDPOINTS'`.

Could you move this this into `_register_view()` and apply it to all endpoints of name schema?

The config lookup should use `get()` instead of `setdefault()`. There is also no need for temp variable here.